scripts/cangen/README.md

```diff
@@ -0,0 +1,41 @@
+# Cangen
+Cangen is a cli package used to generate C++/C files from a corresponding dbc file. This tool automates the can generation process and ensures can generation is consistent between all projects. 
+
+### Installation
+First install the cangen package through the terminal:
+
+```
+pip install racecar/scripts/cangen
+```
+The directory where you run this command does not matter, as long as the path specified leads to the directory containing the pyproject.toml for cangen.
```

Comment:
I didn't mean for my reply to Sam to be included in the README, that was for his knowledge. We should be clear here and provide just a single installation command.

Replace the "Installation" section with

````markdown
## Installation

From the `racecar/` directory, run

```bash
pip install scripts/cangen
```
````

Also notice that I used only 2 `##` before the header instead of 3 `###`. Markdown headers should only increment by one level at a time. You should also fix this in the Usage section